test/__tests__/data/mp_input.json

```diff
@@ -1302,7 +1302,7 @@
       "sentat": "2020-01-24t06:29:02.359z",
       "timestamp": "2020-01-24t11:59:02.402+05:30",
       "type": "group",
-      "userid": "hjikl"
+      "userId": "hjikl"
```

Comment:
Can you add a test case with more than one groupKeyVal?

Reply:
Added